1208-projects/my_projects.feature

```diff
@@ -0,0 +1,54 @@
+Feature: User My Projects
+
+    @2874
+    Scenario Outline: User opens My Projects
+        Given that a User is logged in
+        When they navigate to the My Projects Page
+        Then they see a Create New Project Button
+        And they see a search field
+        And they see a list of Active Projects
+        And they see a collapsed list of Concluded Projects
+        And the lists contains Projects where the User has the role as:
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+            | Participants          |
+        And each Project has:
+            | Title           |
+            | Institution     |
+            | Project Manager |
+        And each Project has an Edit button if the User has "<Role>" 
+        And the list can be sorted by:
+            | Title           |
+            | Institution     |
+            | Project Manager |
+        And they see pagination buttons for the Concluded Projects list
+        Examples:
+            | Role                  |
+            | Project Owner         |
+            | Project Manager       |
+            | Local Project Manager |
+    
+    Scenario: User opens a Project's Landing Page
+        Given User opens My Projects
+        When they click a Project's Title 
+        Then they see the Landing Page for the Project
+
+    Scenario Outline: User Edits a Project in Project Wizard
+        Given User has "<Role>" on the Project
+        When they click the Edit Button 
```

Comment:
Denne sier ikke noe om hvor i appen man er nå
```suggestion
        Given User opens My Projects
        And User has "<Role>" on a Project
        When they click the Edit Button 
```

Reply:
Justerer i egen commit

Reply:
Denne "User has "<Role>" er strengt tatt unødvendig, da Edit knappen kun skal vere her om brukaren har gitt rolle, jamfør linje 20 i det andre scenario. Men det er sikkert greit å gjenta det her, så slipper vi at to scenario har direkte avhengigheter - som kun er implesitt i teksten.